pkg/gadgettracermanager/gadgettracermanager.go

```diff
@@ -161,15 +163,23 @@ func (g *GadgetTracerManager) AddContainer(_ context.Context, containerDefinitio
 	}
 
 	container := containercollection.Container{
-		ID:        containerDefinition.Id,
-		Pid:       containerDefinition.Pid,
-		Namespace: containerDefinition.Namespace,
-		Podname:   containerDefinition.Podname,
-		Name:      containerDefinition.Name,
-		Labels:    map[string]string{},
+		Runtime: containercollection.RuntimeMetadata{
+			BasicRuntimeMetadata: eventtypes.BasicRuntimeMetadata{
+				ContainerID: containerDefinition.Id,
+			},
+		},
+		Pid: containerDefinition.Pid,
+		K8s: containercollection.K8sMetadata{
+			BasicK8sMetadata: eventtypes.BasicK8sMetadata{
+				Namespace:     containerDefinition.Namespace,
+				PodName:       containerDefinition.Podname,
+				ContainerName: containerDefinition.Name,
+			},
+			PodLabels: map[string]string{},
+		},
```

Comment:
For another PR: We need to update the proto container definition to have the k8s and runtimes fields as our golang definition, otherwise this is going to be too confusing.

Reply:
Added to the todo list.